client/me/get-apps/download-card.jsx

```diff
@@ -0,0 +1,82 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import Button from 'components/button';
+
+const DownloadCard = ( { translate } ) => {
+	const trackAppDownloadClick = () => {};
+	const availableLinkTag = ( platformLink ) => {
+		return <a href={ platformLink } onClick={ trackAppDownloadClick } />;
+	};
+
+	const windowsLink = 'https://apps.wordpress.com/d/windows?ref=getapps';
```

Comment:
We can use `SCREAMING_SNAKE_CASE` for constants. so this can be something like `const WINDOWS_APP_LINK = 'https://apps.wordpress.com/d/windows?ref=getapps'`